master/main.go

```diff
@@ -125,8 +167,9 @@ func registerWorker(w http.ResponseWriter, r *http.Request) {
 		util.BadRequest(w, "Error unmarshaling body", err)
 		return
 	}
+	newWorker.Active = true
 	workers = append(workers, &newWorker)
-	fmt.Println("Worker: " + newWorker.Address + " successfully registered!")
+	fmt.Println("Worker: " + newWorker.InstanceId + " successfully registered!")
```

Comment:
Misschien in general voor alle endpoints een JSON response maken?

Reply:
Yes aangepast